# request
---
# response
string[] obj_names  # names of objects
float32[] masses  # masses of objects
float32[] inertias # inertia values for objects
float32[] friction_coeffs  # friction coefficients
float32[] widths  # widths of local AABBs
float32[] heights  # heights of local AABBs